src/config/dynamic-theme-fixes.config

```diff
@@ -16147,6 +16147,13 @@ INVERT
 
 ================================
 
+m.eeworld.com.cn
+
+INVERT
+.logo
```

Comment:
I would use .header-logo-search-login

as there are two black icons on the right too.

And merge everything and 1 PR - just check review in #13123

Reply:
Updated as requested.